### DISCUSSION TITLE
How to move the tabset from right to left in Quarto Dashboard

### DISCUSSION THREAD

**Discussion by JauntyJJS** [quarto-dev/quarto-cli#11544] [Q&A] [answered]
### Description

Hi, 

I have updated Quarto to to 1.6.37 and it seems that for the dashboard tabset, the default settings has moved from left to right. 

Is there a way to move it back to left ? 

````qmd
---
title: "test"
format: dashboard
---

# Page {orientation="rows" scrolling="true"}

## Row - Tabsets

### Column - Tabset1 {.tabset}

```{r}
#| label: Tabset1
#| title: "Tabset1"

print("Tabset1")

```

```{r}
#| label: Tabset2
#| title: "Tabset2"

print("Tabset2")

```
````

**Comment by cscheid** [comment]
This wasn't intentional - we're checking and will get back to you.

**Reply by cscheid** [reply]
The tabset rendering is actually completely broken.

**Reply by cscheid** [reply, marked as answer]
Tracking the bug here: https://github.com/quarto-dev/quarto-cli/issues/11549

We'll fix ASAP and issue a new stable release soon. Thanks for the report!
